web/src/components/alerting/AlertingPage.tsx

```diff
@@ -1,23 +1,51 @@
 import { PageSection, Title } from '@patternfly/react-core';
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
-import { HorizontalNav, useActivePerspective } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  HorizontalNav,
+  NamespaceBar,
+  useActivePerspective,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { MonitoringContext, MonitoringProvider } from '../../contexts/MonitoringContext';
 
-const AlertsPage = React.lazy(
-  () => import(/* webpackChunkName: "AlertsPage" */ '../alerting/AlertsPage'),
+const MpCmoAlertsPage = React.lazy(() =>
+  import(/* webpackChunkName: "AlertsPage" */ './AlertsPage').then((module) => ({
+    default: module.MpCmoAlertsPage,
+  })),
 );
-const SilencesPage = React.lazy(
-  () => import(/* webpackChunkName: "SilencesPage" */ '../alerting/SilencesPage'),
+const McpAcmAlertsPage = React.lazy(() =>
```

Comment:
Maybe the naming should be based on the operator deploying it rather than its functionality as this might easily change in the future, for example adding/removing perspectives.

```suggestion
const CooAlertsPage = React.lazy(() =>
```